docs/tutorials/tabular/tabular-quick-start.ipynb

```diff
@@ -60,8 +67,36 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "data_url = 'https://raw.githubusercontent.com/mli/ag-docs/main/knot_theory/'\n",
-    "train_data = TabularDataset(f'{data_url}train.csv')\n",
+    "data_url = 'https://raw.githubusercontent.com/mli/ag-docs/main/knot_theory'\n",
+    "train_input, test_input = f\"{data_url}/train.csv\", f\"{data_url}/test.csv\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following code block is only necessary when running on JupyterLite. It would load HTTP based dataset, since Pyodide is not able to load remove content directly."
```

Comment:
Ditto to previous comment. We shouldn't sacrifice the simplicity of our tutorials for the sake of supporting XYZ niche user environment. Perhaps we could hide these cells by default, and have 1 line telling the user to expand these cells and run them if they are using jupyterlite.

Reply:
Sure. will leverage pyodide-http package for more seamless integration.